### DISCUSSION TITLE
Can't work with nested component

### DISCUSSION THREAD

**Discussion by theprantadutta** [livewire/livewire#3641] [7. Bugs (archived)]
I am working on a dummy project for my university. I am relatively new to livewire. So this might be a big deal at all.
Here's my [GitHub repo](https://github.com/prantaDutta/library_management_system/tree/issue)

When I am creating my auth form, I decided to create a text-field component that will be nested inside both the login and the register component. Now I am passing the props from login to text-field with the text-field value name and other input props. Now, the text-field component is trying to find the name of the input inside the text-field which is understandable. But I need it to look inside the login.php file. Am I doing anything wrong here or is there anything I don't know?

How can I solve this? If you just run my repo and go to the login or register method, you can understand the problem.

**Comment by joshhanley** [comment]
@prantaDutta you can't access data between parent and child components. You will need to use events to send data between them https://laravel-livewire.com/docs/2.x/events

But I would not be creating Livewire components for things like `text-input`, due to the communication issue above, instead you should be using blade components for that https://laravel.com/docs/8.x/blade#components

Hope this helps!
